### DISCUSSION TITLE
Bulk Import CSV

### DISCUSSION THREAD

**Discussion by antmar904** [philhagen/sof-elk#269] [General]
Hi
Is there a way to bulk import multiple CSV files and put them under the same "index" to be searched?

**Comment by philhagen** [comment]
it would depend on what the CSV files contain and if there is a parser for that format and structure.  ES in general is not intended as a generic CSV importer as you'd have with a spreadsheet.  However, we're trying to move away from CSV in general, as JSON is a more structured and focused format.  However, with JSON (and any format, really), a parser would need to be created to handle/enrich each new input format.
